apps/dcos_net/src/dcos_net_mesos_listener.erl

```diff
@@ -21,6 +21,7 @@
 -type task() :: #{
     name => binary(),
     framework => binary() | {id, binary()},
+    runtime => binary(),
```

Comment:
Could you please use `mesos | docker` atoms as a type for `runtime` field? Atoms consume 1 word, binaries consume 3-6 words + data size [1]. Code with atoms, in general, looks much cleaner. Also when you convert binaries to atoms, it's an additional validation of data from mesos.

If you think that it could be other container types, please consider adding the third atom (e.g. `unknown`).

[1] http://erlang.org/doc/efficiency_guide/advanced.html#memory